modules/services/window-managers/i3-sway/lib/options.nix

```diff
@@ -819,4 +819,11 @@ in {
     description = "Default launcher to use.";
     example = "bemenu-run";
   };
+
+  defaultWorkspace = mkOption {
+    type = types.nullOr types.str;
+    default = null;
+    description =
+      "Has to be the value of the keybinding (of the default Workspace).";
```

Comment:
```suggestion
    description =
      "Has to be the value of the keybinding (of the default Workspace).";
    example = literalExample ''"workspace number 9"'';
```

produces:

![2021-05-15_18-01](https://user-images.githubusercontent.com/16734772/118381386-a6f25f80-b5a7-11eb-86d5-ac3e2b65f951.png)

Reply:
Thx for the hint and explanation :)